### PR TITLE
Adding testing module capable of run with setup.py

### PR DESCRIPTION
**Pull request by hirokgreen**
* test command : python setup.py tests
(make sure to install autoslug)

### CODE REVIEW

File: drf_spirit/tests/settings.py [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+# Application definition
+
+INSTALLED_APPS = [
+    'django.contrib.admin',
+    'django.contrib.auth',
+    'django.contrib.contenttypes',
+    'django.contrib.sessions',
+    'django.contrib.messages',
+    'django.contrib.staticfiles',
+    # drf_spirit
```
Comment (safwanrahman) [inline drf_spirit/tests/settings.py]: Remove the comment

File: drf_spirit/tests/test_models.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+        Category.objects.create(title='alice',)
+        self.client = APIClient()
+
+    def test_get(self):
+        resp = self.client.get('/category/alice/',)
+        self.assertEqual(resp.status_code, 200)
+
+    def test_post(self):
+        resp = self.client.post(
+            '/category/', {'title': 'someone', 'description': 'hello world'}, format='json')
```
Comment (safwanrahman) [inline drf_spirit/tests/test_models.py]: use ``reverse``

File: drf_spirit/tests/urls.py [hunk 3]
```diff
@@ -0,0 +1,9 @@
+from django.conf.urls import url, include
+from django.contrib import admin
+
+from drf_spirit import urls 
+
+urlpatterns = [
+    url(r'^admin/', admin.site.urls),
```
Comment (alaminopu) [inline drf_spirit/tests/urls.py]: Remove `admin` URL from test.

File: runtests.py [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+def runtests():
+    TestRunner = get_runner(settings)
+    test_runner = TestRunner(verbosity=1, interactive=True)
+    if hasattr(django, 'setup'):
+        django.setup()
+    failures = test_runner.run_tests(['drf_spirit'])
+    sys.exit(bool(failures))
+
+if __name__ == '__main__':
+    runtests()
```
Comment (alaminopu) [inline runtests.py]: Add new line at the end of the file.

File: drf_spirit/tests/test_models.py [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+from drf_spirit.models import Category
+
+class CategoryTest(TestCase):
+
+    def setUp(self):
+        Category.objects.create(title='alice',)
+        self.client = APIClient()
+
+    def test_get(self):
+        resp = self.client.get('/category/alice/',)
```
Comment (safwanrahman) [inline drf_spirit/tests/test_models.py]: Here also

File: drf_spirit/tests/test_models.py [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+from rest_framework.test import APIClient
+from drf_spirit.models import Category
+
+class CategoryTest(TestCase):
+
+    def setUp(self):
+        Category.objects.create(title='alice',)
+        self.client = APIClient()
+
+    def test_get(self):
```
Comment (alaminopu) [inline drf_spirit/tests/test_models.py]: @hirokgreen update the test and make sure they are working.

File: drf_spirit/tests/test_category.py [hunk 7]
```diff
@@ -0,0 +1,22 @@
+from django.test import TestCase
+from django.urls import reverse
+from rest_framework.test import APIClient
+from drf_spirit.models import Category
+
+class CategoryTest(TestCase):
+
+    def setUp(self):
+        self.client = APIClient()
+        from django.contrib.auth.models import User
```
Comment (alaminopu) [inline drf_spirit/tests/test_category.py]: Why did you import here?

**Pull request merged**